src/llmq/params.h

```diff
@@ -440,6 +441,33 @@ static constexpr std::array<LLMQParams, 13> available_llmqs = {
         .recoveryMembers = 50,
     },
 
+    /**
+     * llmq_25_67
+     * This quorum is deployed on Testnet and requires
+     * 25 participants
+     *
+     * Used by Dash Platform
+     */
+    LLMQParams{
+        .type = LLMQType::LLMQ_25_67,
+        .name = "llmq_25_67",
+        .useRotation = false,
+        .size = 25,
+        .minSize = 22,
+        .threshold = 67,
```

Comment:
these are absolute values, not percentages. Need to change to probably 20

Reply:
Correct